console/console-init/ui/src/components/index.ts

```diff
@@ -25,7 +25,7 @@ export * from "./FinishedStep";
 export * from "./Editor";
 export * from "./FormAndJson";
 export * from "./InputText";
-export * from "./ProjectHeaderCard";
+export * from "../modules/project/components/ProjectHeaderCard";
```

Comment:
use abosulte path. you can get the files from modules level which is top level folder so don't need to add dots.
export * from "/modules/project/components/ProjectHeaderCard";